src/Writer.jl

```diff
@@ -282,7 +282,7 @@ end
 function show_json(io::SC, s::CS, x::CompositeTypeWrapper)
     begin_object(io)
     for fn in x.fns
-        show_pair(io, s, fn, getfield(x.wrapped, fn))
+        show_pair(io, s, fn, getproperty(x.wrapped, fn))
```

Comment:
Just `x.wrapped.fn`?

Reply:
That wouldn't work here. `fn` is a variable whose value is a `Symbol` that is the property name. It is not itself the property name.

Reply:
Oh sorry, I missed the context.